api/app.js

```diff
@@ -78,6 +78,13 @@ app.use('/data', dsRouter);
 app.use(['/payments', '/payment'], stripeRouter);
 app.use(['/calendar-events'], calendarEventsRouter);
 app.use('/getting_started', gettingStartedRouter);
+
+app.use('*', errorhandler);
+//eslint-disable-next-line
+function errorhandler(err, req, res, next) {
+  res.status(err.status || 500).json(err.message);
+}
+
```

Comment:
Nice work with the errorhandler function! I see you brought it over from the childrenRouter.js file to the app.js so that it covers all the routes. 
Now you get 'missing idToken' for a message instead of 'not found'. 

Great job making the error from a missing jwt meaningful.